introduction_to_amazon_algorithms/xgboost_abalone/xgboost_parquet_input_training.ipynb

```diff
@@ -21,6 +21,7 @@
     "import os\n",
```

Comment:
<p><code>FILE_NAME</code>could be more accurate? And we can replace Line 20 with the constant as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good catch, thanks Peter!